anaconda.spec.in

```diff
@@ -286,9 +286,19 @@ Requires: skopeo
 Requires: nvme-cli
 # Needed for bootc
 Requires: podman
+Requires: bootc
+Requires: bootupd
```

Comment:
I think we really should set a minimum version for bootc there to make sure we are bug compatible with it & don't end up with an older version that lacks the latest behavior we just adjusted our code to.

Won't really help with newer version breaking things, but I don't think we can really version lock it like that - best setup some sort of reverse depedency testing on the bootc side to prevent new bootc releases breaking Anaconda/Fedora due to regressions once bootc support is merged.

Reply:
I think time will tell how much we should be bounded to a particular bootc version. Right now I would leave it be and fix the version if needed in the future.